src/routes/trip.route.js

```diff
@@ -10,11 +10,11 @@ import UserValidation from '../validations/user.validation';
 
 const router = express.Router();
 
-router.post('/one-way-trips', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, oneWayTripValidation, TripController.requestOneWayTrip); // One way trip route
-router.post('/return-trip', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, TripValidation, TripController.requestReturnTrip); // Return trip route
-router.get('/requests/:userId', authMiddleware.checkUserLoggedIn, requestValidation, TripController.userTripRequestList); // user request list route
-router.post('/multi-city-trips', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, multiCityTripValidation, TripController.requestMultiCityTrip);
+router.post('/trips/one-way-trip', authMiddleware.checkUserLoggedIn, authMiddleware.checkIfUserHaveManager, oneWayTripValidation, TripController.requestOneWayTrip); // One way trip route
```

Comment:
This feedback is not properly implemented. Refer to https://github.com/andela/the_spinners-backend/pull/40#discussion_r366213513